tracer/src/Datadog.AutoInstrumentation.NativeLoader/com_ptr.h

```diff
@@ -0,0 +1,186 @@
+﻿#ifndef DD_CLR_PROFILER_COM_PTR_H_
```

Comment:
This file already exists in  `shared/src/native-src/com_ptr.h`
Do you have any specific needs?

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a